### ISSUE TITLE
Error during package load

### ISSUE THREAD

**Issue by allengreer-latero** [emacs-lsp/lsp-origami#11]
**Emacs version**:
GNU Emacs 27.1 (build 2, x86_64-apple-darwin19.6.0, NS appkit-1894.60 Version 10.15.6 (Build 19G2021)) of 2020-08-14

**Spacemacs version**:
Spacemacs v.0.300.0 (commit 2fbd853f9)

**Output in *Messages* Buffer**
Error processing message (error "Autoloading file `.../emacs.d/elpa/27.1/develop/lsp-origami-20200914.1846/lsp-origami.elc` failed to define function lsp-origami").

Is this something local to the package, or should I open up an issue with Spacemacs?ew

**Comment by AntonOrlovAmbulnz** [comment]
Have same in `spacemacs` but with emacs 28

**Comment by oxyum** [comment]
Fixed in syl20bnr/spacemacs#13959

**Comment by jmromer** [comment]
Still seeing this on develop @ 14dcebd4a30a1a18847ad:
```
File mode specification error: 
(error Autoloading file ~/.emacs.d/elpa/28.0/develop/lsp-origami-20200914.1846/lsp-origami.el failed to define function lsp-origami)
```
